cheroot/server.py

```diff
@@ -467,7 +467,8 @@ def _fetch(self):
             chunk_size = line.pop(0)
             chunk_size = int(chunk_size, 16)
         except ValueError:
-            raise ValueError('Bad chunked transfer size: ' + repr(chunk_size))
+            raise ValueError('Bad chunked transfer size: {}'
```

Comment:
Just use
```suggestion
            raise ValueError('Bad chunked transfer size: {!r}'
```
instead of calling `repr()` manually.